microsoft-edge/webview2/how-to/remote-debugging.md

```diff
@@ -101,6 +101,36 @@ Attach Microsoft Edge DevTools remotely to a WebView2 WinUI 2 (UWP) app as follo
 Now you can use Microsoft Edge DevTools to inspect and debug the WebView2 control.
 
 
+<!-- ====================================================================== -->
+## Debugging Hololens 2 WebView2 WinUI 2 (UWP) apps
+
+Attach Microsoft Edge DevTools remotely to a Hololens 2 WebView2 WinUI 2 (UWP) app as follows:
+
+1. Follow steps 1 & 2 under [Debugging desktop WebView2 WinUI 2 (UWP) apps](#debugging-desktop-webview2-winui-2-uwp-apps).
+
+1. Enable Device Portal: Open Windows **Settings**, select **Update & Security** > **For developers**, and then turn on the **Developer Portal** toggle.

```

Comment:
within any list, either have 1 or 2 spaces after period.  In a list, the indent amount of a png must match that of the text.  I can check it out in VS Code locally and post changes if needed.  Or do a PR into this PR.